dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/export/event/JdbcEventStore.java

```diff
@@ -1664,13 +1664,14 @@ private String getOrgUnitSql( User user, EventSearchParams params, String ouTabl
             return getAccessibleOrgUnitsPath( params, user, ouTable );
         }
 
-        return switch (orgUnitSelectionMode) {
-            case ALL -> null;
-            case CHILDREN -> getChildrenOrgUnitsPath(params, ouTable);
-            case DESCENDANTS -> getDescendantOrgUnitsPath(params, ouTable);
-            case CAPTURE -> getCaptureOrgUnitsPath(user, ouTable);
-            case SELECTED -> getSelectedOrgUnitsPath(params, ouTable);
-            default -> getAccessibleOrgUnitsPath(params, user, ouTable);
+        return switch ( orgUnitSelectionMode )
```

Comment:
all of the switch statements\expressions in this PR look like they are missing indentation. The `case`s and `default` should be indented once from the method curly brackets to indicate that they are enclosed/part of that method. This would make it consistent with how other formatted methods look.